app/models.py

```diff
@@ -252,6 +252,10 @@ def is_anonymous_requester(self):
     def get_id(self):
         return USER_ID_DELIMITER.join((self.guid, self.auth_user_type))
 
+    @property
+    def name(self):
+        return "{} {}".format(self.first_name, self.last_name)
```

Comment:
forgot to point out yesterday, this can just be `return ' '.join((self.first_name, self.last_name))`